crates/volta-core/src/tool/package/mod.rs

```diff
@@ -59,15 +59,15 @@ impl Package {
         Package { name, details }
     }
 
-    fn fetch_internal(&self) -> Fallible<()> {
+    fn fetch_internal(&self, mut session: &mut Session) -> Fallible<()> {
```

Comment:
I believe the `mut` prefix before the variable name is extraneous:

```suggestion
    fn fetch_internal(&self, session: &mut Session) -> Fallible<()> {
```